packages/athena/libs/ca_lib.js

```diff
@@ -303,15 +312,18 @@ module.exports = (logger, ev, t) => {
 				certificate_request: t.key_lib.generateCSR('CN=' + fabricCAClient.enrollmentID, pair.prvKeyObj)
 			};
 
+			const path = fabricCAClient.baseAPI + 'reenroll';
+			const method = 'POST';
+
 			// get the auth token (csr)
-			const auth_token = t.fabric_utils.generateAuthToken(reenrollCSR, t.signingIdentity);
```

Comment:
I assume that passing t.signingIdentity was a leftover from an old implementation, so I've removed it. The current generateAuthToken() implementation already accesses t.signingIdentity internally.